airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/PostgresQueryUtils.java

```diff
@@ -68,14 +69,16 @@ public class PostgresQueryUtils {
    * value of the xmin snapshot (which is a combination of 1 and 2). If no wraparound has occurred,
    * this should be the same as 2.
    */
-  public static void logXminStatus(final JdbcDatabase database) throws SQLException {
+  public static XminStatus getXminStatus(final JdbcDatabase database) throws SQLException {
     LOGGER.debug("xmin status query: {}", XMIN_STATUS_QUERY);
     final List<JsonNode> jsonNodes = database.bufferedResultSetQuery(conn -> conn.prepareStatement(XMIN_STATUS_QUERY).executeQuery(),
```

Comment:
Can there be other system columns that may be useful even for just logging

Reply:
Which ones were you thinking of? We can always add more here later if you come up with them afterwards